cmd/documentation/exchange_templates/exchange_quickdata_readme.tmpl

```diff
@@ -0,0 +1,135 @@
+{{define "exchange quickdata" -}}
+{{template "header" .}}
+## Overview
+
+This package provides a means to quickly request and receive data for an individual exchange, asset and currency pair. For the times you just really want to get some data without config and minimal setup.
```

Comment:
🦍 Optional pedantic nitpick:
```suggestion
This package provides a means to quickly see data for an individual exchange, asset and currency pair. For the times you just really want to get some data without config and minimal setup.
```

Reply:
Fair to hate on request and receive, but "see" is too casual. I could accept it for the main.go readme, since you are seeing the output, but you are receiving proper structs that you can do things with. So I'll change to `quickly encromulate data` or something else

Reply:
🦍 is easily placated - resolving is fine.
But `grab, `review`, `stream`.

Thought actually .. just `request` seems fine.